tests/resources/pubsub_topics.nim

```diff
@@ -0,0 +1,12 @@
+proc getPubsunTopic*(contentTopicName: string): string =
+  return "/waku/2/$contentTopicName"
```

Comment:
I think pubsub is mispelled in the proc's name.

And also believe that the `contentTopicName` parameter is kind of misnamed. Content topic is not part of the pubsub topic name

Reply:
Ah, yes, oversight, thanks!